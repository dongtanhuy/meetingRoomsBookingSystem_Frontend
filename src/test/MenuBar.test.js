/**
 * @jest-environment node
 */
/* global it, describe, beforeEach, before, after, afterEach, expect */
/* eslint no-unused-vars: "off" */
import React from 'react'
import { shallow } from 'enzyme'
import MenuBar from '../components/MenuAppBar'

describe('Menu Bar', () => {
  it('It should render menu bar', () => {
    const menuBar = shallow(<MenuBar />)
    expect(menuBar.getElements()).toMatchSnapshot()
  })
})
