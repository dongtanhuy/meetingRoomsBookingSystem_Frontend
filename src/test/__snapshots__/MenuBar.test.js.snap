// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Bar It should render menu bar 1`] = `
Array [
  <MenuAppBar
    classes={
      Object {
        "root": "MenuAppBar-root-1",
      }
    }
  />,
]
`;
